From 76be89ce1fb815f7ebeeed9bfeb097b4556eaf11 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Tue, 17 Jan 2023 11:50:27 +0100
Subject: [PATCH] transmission: do not prevent service from running

Enable transmission in /etc/config by default to make sure when we
enable the service, transmission can run without having to manually
enable it in the config.

Users, by default, expect that the service will run automatically and that
they will configure it while the service is running to see what was changed
and what is the behavior of each function there. This change is going to enable
transmission to run after installing it instead of tweaking the config manually
(in LuCI or by hand). This is somehow different than how OpenWrt does it. They
force users to configure the app before running. Once they are confident that
the configuration looks good, they enable it, which is also good behavior.
Still, we need to improve the user experience and somehow make it different as
we want that even non-IT guys/girls can configure what they need and install
what they need in our simplified interface called (re)Foris, where we provided
list of packages, which they can install and if it does not run by default, it
looks like something is wrong. For transmission, we also provide
Transmission WebApp, the integration for Turris OS, which you can find on the
landing page and easily click on it and start configuring it.
---
 net/transmission/files/transmission.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/transmission/files/transmission.config b/net/transmission/files/transmission.config
index 770908c7f2..88795014ef 100644
--- a/net/transmission/files/transmission.config
+++ b/net/transmission/files/transmission.config
@@ -1,5 +1,5 @@
 config transmission
-	option enabled 0
+	option enabled 1
 	option config_dir '/tmp/transmission'
 	option config_overwrite '1'
 	option user 'transmission'
-- 
2.39.1

